hswidget/dc6widget/widget.go

```diff
@@ -117,13 +124,14 @@ func (p *widget) makeViewerLayout() giu.Layout {
 			imgui.EndGroup()
 		}),
 		giu.Separator(),
-		// NOTE: most of DC6 animations arent `playable`
-		// they are sprites, however exists some animations
-		// for example monsters' animations, where `play`
-		// feature is useful
 		p.makePlayerLayout(viewerState),
 		giu.Separator(),
 		widget,
+		giu.Separator(),
+		giu.Button("Tiled View##"+p.id+"mergebutton").Size(buttonW, buttonH).OnClick(func() {
```

Comment:
for coherence, change `mergebutton` to `tiledButton

Reply:
fixed